tests/capice/cli/test_args_handler_predict.py

```diff
@@ -0,0 +1,217 @@
+import unittest
+
+import pickle
+import os
+from io import StringIO
+
+from pathlib import Path
+from unittest.mock import patch
+
+from molgenis.capice.cli.args_handler_predict import ArgsHandlerPredict
+from argparse import ArgumentParser
+
+
+_project_root_directory = Path(__file__).absolute().parent.parent.parent.parent
+
+
+class TestArgsHandlerPredict(unittest.TestCase):
```

Comment:
I would love to see a comment string above the test (or a docstring in the test) that would describe the test. Like for instance `test_model_pypi_invalid_prerelease_name` would have documentation like "Tests if model_version 1.0.0pre1 raises error with capice_version 1.0.0". It doesn't have to describe the whole thing like imputs and expectations, just some note of what the function tests so that we don't get confused in the future when we look back at these tests.